src/javascript/SelectorTypes/Picker/configs/ContentPickerConfig.js

```diff
@@ -41,6 +41,7 @@ export const ContentPickerConfig = {
         dialogTitle: 'content-editor:label.contentEditor.edit.fields.contentPicker.modalTitle',
         searchPlaceholder: 'content-editor:label.contentEditor.edit.fields.contentPicker.searchPlaceholder',
         displayTree: true,
-        displaySiteSwitcher: true
+        displaySiteSwitcher: true,
+        itemSelectionAdapter: content => content ? content.uuid : null
```

Comment:
Will we need an adapter if MediaPickerConfig as well?

Reply:
yes, right. I don't know why we need that exactly, but it was here in the previous picker

Reply:
Sounds good to me!